src/Compilers/CSharp/Test/Emit2/Semantics/ParamsCollectionTests.cs

```diff
@@ -5198,24 +5208,14 @@ static string generateMethodSignature(string methodName, string parameterType) =
         [InlineData("System.ReadOnlySpan<int>", "System.Span<int>", "System.ReadOnlySpan<System.Int32>")]
         [InlineData("System.ReadOnlySpan<int>", "System.Span<object>", "System.ReadOnlySpan<System.Int32>")]
         [InlineData("System.ReadOnlySpan<int>", "System.Span<int?>", "System.ReadOnlySpan<System.Int32>")]
-
-        // Ambiguous for inline collection expression, but 'int' is a better conversion target than 'object' in params case
         [InlineData("System.ReadOnlySpan<object>", "System.Span<int>", "System.Span<System.Int32>")] // cannot convert object to int
```

Comment:
>// cannot convert object to int [](http://example.com/codeflow?start=101&length=31)

Do we still need these "// cannot convert ..." comments?

Reply:
No, I meant to remove them, thanks.